Sources/AppStoreConnectCLI/Model/BetaGroup.swift

```diff
@@ -45,6 +46,19 @@ struct BetaGroup: TableInfoProvider, ResultRenderable, Equatable {
     }
 }
 
+extension BetaGroup {
+    init(extendedBetaGroup: ExtendedBetaGroup) {
+        app = App(extendedBetaGroup.app)
+        groupName = extendedBetaGroup.betaGroup.attributes?.name
+        isInternal = extendedBetaGroup.betaGroup.attributes?.isInternalGroup
+        publicLink = extendedBetaGroup.betaGroup.attributes?.publicLink
+        publicLinkEnabled = extendedBetaGroup.betaGroup.attributes?.publicLinkEnabled
+        publicLinkLimit = extendedBetaGroup.betaGroup.attributes?.publicLinkLimit
+        publicLinkLimitEnabled = extendedBetaGroup.betaGroup.attributes?.publicLinkLimitEnabled
+        creationDate = extendedBetaGroup.betaGroup.attributes?.createdDate?.formattedDate
+    }
```

Comment:
will this be better?
```suggestion
   init(app: App, attributes: AppStoreConnect_Swift_SDK.BetaGroup.Attributes?) {
        self.app = app
        groupName = attributes?.name
        isInternal = attributes?.isInternalGroup
        publicLink = attributes?.publicLink
        publicLinkEnabled = attributes?.publicLinkEnabled
        publicLinkLimit = attributes?.publicLinkLimit
        publicLinkLimitEnabled = attributes?.publicLinkLimitEnabled
        creationDate = attributes?.createdDate?.formattedDate
    }

    init(extendedBetaGroup: ExtendedBetaGroup) {
        self.init(app: App(extendedBetaGroup.app), attributes: extendedBetaGroup.betaGroup.attributes)
    }
```

Reply:
🤔 I think I would make that change @DechengMa if I needed to call: `init(app: App, attributes: AppStoreConnect_Swift_SDK.BetaGroup.Attributes?)` directly
Since I'm only calling: `init(extendedBetaGroup: ExtendedBetaGroup)` I think it's fine to not break it apart